packages/appstore/src/components/modals/wallet-modal/wallet-modal.tsx

```diff
@@ -1,19 +1,44 @@
 import React from 'react';
 import { Modal } from '@deriv/components';
+import { useCurrencyConfig, useWalletList } from '@deriv/hooks';
+import { getCurrencyDisplayCode } from '@deriv/shared';
+import { localize } from '@deriv/translations';
 import WalletModalHeader from './wallet-modal-header';
 import WalletModalBody from './wallet-modal-body';
 import { observer, useStore } from '@deriv/stores';
+import type { TWalletType } from './provider';
+
+// TODO: remove this type when we can use real current wallet object
+export type TWallet = Omit<Exclude<ReturnType<typeof useWalletList>['data'], undefined>[0], 'balance'> & {
+    balance?: string | number;
```

Comment:
Replace `balance` type with proper one for current implementation

Reply:
@heorhi-deriv What do you mean by current implementation? 🤔 Are we changing the type somewhere? 🤔